src/funding/venmo/style.scoped.scss

```diff
@@ -1,4 +1,7 @@
 
+.app-label {
+    margin-left: 0.25rem;
```

Comment:
Looks like it needs to be positioned down a bit as well.

Reply:
I think you're right, going to take this back with design because the result was approved but it looks higher than the venmo logo